apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/helpers.php

```diff
@@ -208,3 +208,51 @@ function populate_wp_template_data() {
 }
 register_activation_hook( __FILE__, __NAMESPACE__ . '\populate_wp_template_data' );
 add_action( 'switch_theme', __NAMESPACE__ . '\populate_wp_template_data' );
+
+function has_legacy_FSE_template_edits( $blog_id ) {
+	switch_to_blog( $blog_id );
+	$theme_slug = normalize_theme_slug( get_stylesheet() );
+
+	dangerously_load_full_site_editing_files();
+
+	// Get saved template part markup
+	$template_manager = new WP_Template();
+	$header_content = $template_manager->get_template_content( 'header' );
+	$footer_content = $template_manager->get_template_content( 'footer' );
```

Comment:
This is the part that isn't working currently.

Digging into these methods more, it seems to be failing due to this taxonomy call:

https://github.com/Automattic/wp-calypso/blob/a99cc0181a1f566053021777961b4014e22b5c14/apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/templates/class-wp-template.php#L79

This taxonomy seems to be used to store the postID of the headers and footer posts saved for various themes, but in this context it is returning `false`. However, this seems to work just find when loading the editor for that site. 🤔

Reply:
Ok, this part works now after ensuring the post type is registered.